### PR TITLE
Vox slave implant - finale version

### PR DESCRIPTION
**Pull request by Rayfod**
But I still cannot add shit to the vox map

Adds a vox slave implant
Added the prompt, added a lockbox for it all
Did a bunch of coding

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm [hunk 1]
```diff
@@ -13,8 +13,8 @@ Small, little HP, poisonous.
 	response_disarm = "gently pushes aside"
 	response_harm   = "stamps on"
 	destroy_surroundings = 0
-	health = 15
-	maxHealth = 15
+	health = 50
```
Comment (Anton-Kr) [inline code/modules/mob/living/simple_animal/hostile/voxslug.dm]: Putting unrelated changes in a PR that doesn't mention them is bad. If you want vox slugs buffed, make a seperate PR. The faction fix is however fine.

**Review by Anton-Kr — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm (hunk 1 above)
Comment (Rayfod) [reply code/modules/mob/living/simple_animal/hostile/voxslug.dm]: Ah sure, I asked Carl beforehand if it was okay which is why I took it in.

**Comment by Rayfod** [comment]
**Please do not merge this PR in the next 24 hours**

Waiting for #1408 to be added as well

**Comment by Rayfod** [comment]
Yeah, my fork is still suffering since I did not get the map update from #1408, so I will most likely have to close this PR as well.

**Comment by Anton-Kr** [comment]
Can always add it to map in separate PR, up to you.

**Pull request closed without merging**